deploy/helm/templates/beyla/configmap.yaml

```diff
@@ -39,6 +39,8 @@ data:
         - k8s_namespace: "*"
       exclude_instrument:
         - exe_path: "{*otelcol*|*beyla*}"
+        - k8s_pod_annotations:
+            sw.ebpf.instrumentation/exclude: "true"
```

Comment:
Do we need this annotation? Should it be annotation or label? Is exclude_instrument working as expected here?